lib/Cake/Utility/CakeTime.php

```diff
@@ -1178,7 +1177,7 @@ protected static function _strftime($format, $timestamp) {
 				$valid = Multibyte::checkMultibyte($format);
 			}
 			if (!$valid) {
-				$format = mb_convert_encoding($format, 'UTF-8', 'ISO-8859-1');
+				$formatted = mb_convert_encoding($format, 'UTF-8', 'ISO-8859-1');
```

Comment:
Hi @kamilwylegala, I'm curious about the change in this line. I see that instead of updating the value of `$format`, you're instantiating a new variable `$formatted`. But it doesn't seem like `$formatted` gets used anywhere, and then the method just returns `$format`. What am I missing?

Reply:
Oh good deal - glad I mentioned it. And thanks for finding an easy(ish) solution!